resources/views/components/dropdown.blade.php

```diff
@@ -1,4 +1,4 @@
-@props(['align' => 'right', 'width' => '48', 'contentClasses' => 'py-1 bg-white', 'dropdownClasses' => ''])
+@props(['align' => 'right', 'widthClass' => 'w-48', 'contentClasses' => 'py-1 bg-white', 'dropdownClasses' => ''])
```

Comment:
Ordinarily, I think this would be a breaking change because the Blade components in Jetstream aren't published by default. However, in this case, the prop wasn't actually doing anything so I think it's okay.